test/saml11.tests.js

```diff
@@ -1,127 +1,195 @@
+        "3464c5bdd2be7f2b6112e2f08e9c0024e33d9fe2",
+      ], // WRONG thumbprints
+      realm: "spn:408153f4-5960-43dc-9d4f-6b717d772c8d",
+      checkRecipient: false,
+      checkExpiration: false,
+    }); // dont check expiration since we are harcoding the token
+    var profile = saml_passport.validateSamlAssertion(
+      signedAssertion,
+      function (error, profile) {
+        assert.equal(
+          "Invalid thumbprint (configured: 3464C5BDD2BE7F2B6112E2F08E9C0024E33D9FE1, 3464C5BDD2BE7F2B6112E2F08E9C0024E33D9FE2. calculated: 3464C5BDD2BE7F2B6112E2F08E9C0024E33D9FE0)",
+          error.message
+        );
+        done();
+      }
+    );
   });
 
-  it('should fail when the X509Certificate is invalid', function (done) {
-    const signedAssertion = fs.readFileSync(__dirname + '/samples/plain/samlresponse_saml11_invalid_cert.txt').toString();
+  it("should fail when the X509Certificate is invalid", function (done) {
+    const signedAssertion = fs
+      .readFileSync(
+        __dirname + "/samples/plain/samlresponse_saml11_invalid_cert.txt"
+      )
+      .toString();
     const options = {
       checkDestination: false,
-      thumbprint: '119B9E027959CDB7C662CFD075D9E2EF384E445F'
+      thumbprint: "119B9E027959CDB7C662CFD075D9E2EF384E445F",
     };
 
     const saml_passport = new SamlPassport(options);
-    saml_passport.validateSamlAssertion(signedAssertion, function(err, profile) {
-    const errorMessages = [
-        'The signing certificate is invalid (PEM_read_bio_PUBKEY failed)',
-        'The signing certificate is invalid (error:0906700D:PEM routines:PEM_ASN1_read_bio:ASN1 lib, error:0D07803A:asn1 encoding routines:ASN1_ITEM_EX_D2I:nested asn1 error, error:0D068066:asn1 encoding routines:ASN1_CHECK_TLEN:bad object header)',
-        'The signing certificate is invalid (error:0D07803A:asn1 encoding routines:asn1_item_embed_d2i:nested asn1 error, error:0D068066:asn1 encoding routines:asn1_check_tlen:bad object header)'
-      ];
-
-      assert.ok(err, 'The signing certificate was unexpectedly valid');
-      assert.ok(/signing certificate is invalid/.test(err.message), 'Error message is not the default invalid message');
-      assert.ok(errorMessages.includes(err.message), 'Error message for invalid certificate is incorrect');
-
-      done();
-    });
+    saml_passport.validateSamlAssertion(
+      signedAssertion,
+      function (err, profile) {
+        const errorMessages = [
+          "The signing certificate is invalid (PEM_read_bio_PUBKEY failed)",
+          "The signing certificate is invalid (error:0906700D:PEM routines:PEM_ASN1_read_bio:ASN1 lib, error:0D07803A:asn1 encoding routines:ASN1_ITEM_EX_D2I:nested asn1 error, error:0D068066:asn1 encoding routines:ASN1_CHECK_TLEN:bad object header)",
+          "The signing certificate is invalid (error:0688010A:asn1 encoding routines::nested asn1 error, error:06800066:asn1 encoding routines::bad object header)",
```

Comment:
There are bunch of auto formatting changes. But the gist of the change is changing error message from 

` 'The signing certificate is invalid (error:0D07803A:asn1 encoding routines:asn1_item_embed_d2i:nested asn1 error, error:0D068066:asn1 encoding routines:asn1_check_tlen:bad object header)'
`
to

`'The signing certificate is invalid (error:0688010A:asn1 encoding routines::nested asn1 error, error:06800066:asn1 encoding routines::bad object header)'`